cedar-policy/src/api.rs

```diff
@@ -2746,7 +2746,7 @@ impl Policy {
     /// conversion to AST is lossy. ESTs for policies generated by this method
     /// will reflect the AST and not the original policy syntax.
     #[cfg_attr(not(feature = "partial-eval"), allow(unused))]
-    fn from_ast(ast: ast::Policy) -> Self {
+    pub fn from_ast(ast: ast::Policy) -> Self {
```

Comment:
Highlighting this change.

Reply:
I don't think we should do this change. `ast::Policy` is a type we don't want to expose.  Is `pub(crate)` sufficient?

Reply:
`pub(crate)` should work since these are unit test instead of integration.